src/toolchain/export-android.md

```diff
@@ -0,0 +1,112 @@
+<!--
+ ~ Copyright (c) godot-rust; Bromeon and contributors.
+ ~ This Source Code Form is subject to the terms of the Mozilla Public
+ ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+ ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Export to Android
+
+```admonish note title="Android CLI and JDK 17"
+Keep track of where you install the Android Command Line Tools and JDK 17
+```
+
+Exporting with gdext for Godot requires some of the same pieces that are required for building Godot from source.
+Specifically, the Android SDK Command Line Tools and JDK 17 as mentioned in Godot's documentation
+[here](https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#requirements).
+
+Once you have those installed, you then need to follow Godot's instructions for setting up the build system
+[here](https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#setting-up-the-buildsystem).
+
+As of writing this, it is not entirely clear if JDK 17 is required in order for Android Command Line Tools to function properly.
+It has not been not tested without it. To be safe, it is included in the documentation for now.
```

Comment:
Is there a way to clarify this? Google doesn't contain installation instructions?

I'd rather avoid users needing to install extra toolchains or higher versions of Java if those turn out to be unnecessary.

Reply:
It looks like there stance is:

> Make sure to choose a JDK version that is higher than or equal to the JDK versions used by plugins that you use in your Gradle build. To determine the minimum required JDK version for the Android Gradle Plugin (AGP), see the compatibility table in the [release notes](https://developer.android.com/build/releases/gradle-plugin).
>
> For example, the Android Gradle Plugin version 8.x requires JDK 17. If you try to run a Gradle build that uses it with an earlier version of the JDK, it reports a message like:

https://developer.android.com/build/jdks

So I guess we have to reference what godot is using.  Lets see if we can find it.

Found it!

https://github.com/godotengine/godot/blob/master/platform/android/java/app/config.gradle

Looks like they have godot configured for java version 17.  I'll update the docs to reference the version godot is using.

Reply:
Very nice, thanks a lot!